emcc.py

```diff
@@ -1886,21 +1900,23 @@ def do_minify(): # minifies the code. this is also when we do certain optimizati
            '--mapFileBaseName', target,
            '--offset', str(0)])
 
-    # Move final output to the js target
-    shutil.move(final, js_target)
```

Comment:
It just feels very... wrong to have a single method do both things. If we decide we do want the copy, we'd need to modify the call, for example - it mixes concerns. It's much simpler when each method does one simple thing.

If all this mixing can save is a move, isn't that almost free anyhow? The OS just renames it.